custom-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
@nizarhandal Please refer exisitng connectors in the repo and prepare readme.md file for your connector.